tiatoolbox/utils/visualization.py

```diff
@@ -573,12 +592,84 @@ def __init__(
         self.zoomed_out_strat = zoomed_out_strat
         self.secondary_cmap = secondary_cmap
         self.blur_radius = blur_radius
+        self.function_mapper = function_mapper
         if blur_radius > 0:
             self.blur = ImageFilter.GaussianBlur(blur_radius)
             self.edge_thickness = 0
         else:
             self.blur = None
 
+    @staticmethod
+    def decode_geom(geom, geom_type):
```

Comment:
```suggestion
    def decode_wkb(wkb, geom_type):
```

Maybe makes it more clear.